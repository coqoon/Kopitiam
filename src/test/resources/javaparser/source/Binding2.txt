class Foo {
  void foo () {
    int a = 20, b = 10;
  }
}
