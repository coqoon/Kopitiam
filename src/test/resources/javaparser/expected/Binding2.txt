class Foo {
  void foo () {
    int a = 20;
    int b = 10;
  }
}
